test/helpers/data_test.go

```diff
@@ -17,16 +17,49 @@ limitations under the License.
 package helpers
 
 import (
-	"fmt"
 	"regexp"
+	"testing"
 )
 
 var matcher = regexp.MustCompile("abcd-[a-z]{8}")
 
-func ExampleAppendRandomString() {
+func TestAppendRandomString(tst *testing.T) {
 	const s = "abcd"
 	t := AppendRandomString(s)
 	o := AppendRandomString(s)
-	fmt.Println(matcher.MatchString(t), matcher.MatchString(o), o != t)
-	// Output: true true true
+	if !matcher.MatchString(t) || !matcher.MatchString(o) || o == t {
+		tst.Fatal()
+	}
+}
+
+func TestMakeK8sNamePrefix(tst *testing.T) {
+	testCases := map[string]string{
```

Comment:
Use the table driven tests format

tc := struct {
want string
got string 
}{ 
{"abcd123": "abcd123"},
...
}